pkg/quarterdeck/auth.go

```diff
@@ -128,12 +132,18 @@ func (s *Server) Register(c *gin.Context) {
 			ProjectID: projectID,
 		}
 		if err = op.Save(ctx); err != nil {
-			log.Error().Err(err).Msg("could not link organization to project")
+			c.Error(err)
 			c.JSON(http.StatusInternalServerError, api.ErrorResponse("could not process registration"))
 			return
 		}
 	}
 
+	// increment registered users (can happen in this function and via invite)
+	metrics.Registered.WithLabelValues("quarterdeck").Inc()
```

Comment:
Not really related to this PR but should `quarterdeck` be a constant somewhere?

Reply:
yes, it would probably have to go into the `metrics` package.